docs/source/markdown/podman-create.1.md

```diff
@@ -562,6 +562,7 @@ Valid values are:
 - `ns:<path>`: path to a network namespace to join
 - `private`: create a new namespace for the container (default)
 - `slirp4netns[:OPTIONS,...]`: use slirp4netns to create a user network stack.  This is the default for rootless containers.  It is possible to specify these additional options:
+  - **cidr=CIDR**: Specify ip range to use for this network. (Default is `10.0.2.0/24`).
```

Comment:
This should probably go one line down (alphabetical order)

Reply:
Moved.